CHANGELOG.md

```diff
@@ -33,6 +33,9 @@ Please refer to the [Process Agent 6.3.0 tag](https://github.com/DataDog/datadog
 * [SANITY] Packaging/Build: Explicilty pin urllib to avoid inadverted updated. See [#3746][]
 * [OTHER] APM enabled by default. See [#3749][] 
 
+### Other Notes
+* Our embedded python unicode interpreter is now UCS4.
```

Comment:
Can you move this with the other `Changes` above, with an `[OTHER]` label, and link to https://github.com/DataDog/omnibus-software/pull/191 for reference (with some markdown `[omnibus-software-191](https://github.com/DataDog/omnibus-software/pull/191)`)